installers/binstall/scripts/nix/install.sh

```diff
@@ -154,6 +157,12 @@ err() {
     exit 1
 }
 
+need_glibc() {
+    if ! check_cmd "/lib/x86_64-linux-gnu/libc.so.6"
+    then err "could not link against 'glibc'"
```

Comment:
Do you think we could also add `Do you have glibc installed?` as part of this error? or even `Do you have glibc >= 2.7 installed?` (I think it's >= 2.7 that we need)